src/Core/Content/Media/MediaHydrator.php

```diff
@@ -0,0 +1,79 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Content\Media;
+
+use Shopware\Core\Framework\Context;
+use Shopware\Core\Framework\DataAbstractionLayer\Dbal\EntityHydrator;
+use Shopware\Core\Framework\DataAbstractionLayer\Entity;
+use Shopware\Core\Framework\DataAbstractionLayer\EntityDefinition;
+use Shopware\Core\Framework\Log\Package;
+use Shopware\Core\Framework\Uuid\Uuid;
+
+#[Package('buyers-experience')]
+class MediaHydrator extends EntityHydrator
```

Comment:
Can you also provide a unit test for this one?

Reply:
There are currently no tests for specific Hydrator classes, neither for product, category, ...
The Hydrator classes are the ones that are implicitly tested with nearly every other test. If there is something wrong with the hydrator, many other tests would fail.